group-coordinator/src/main/java/org/apache/kafka/coordinator/group/runtime/CoordinatorLoader.java

```diff
@@ -53,12 +53,16 @@ public short unknownType() {
     class LoadSummary {
         private final long startTimeMs;
         private final long endTimeMs;
+        private final long totalTimeMs;
+        private final long schedulerTimeMs;
```

Comment:
I wonder if we could find a better name here. They don't really describe well what they are. We could perhaps use startTimeMs, startLoadingTimeMs, endTimeMs. I am open to other suggestions too.

Reply:
how's schedulerQueueTimeMs?

Reply:
that works.